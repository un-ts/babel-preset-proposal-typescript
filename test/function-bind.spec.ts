import _ from './function-bind.js'
import { execute } from './helpers.js'

const proposal = 'function-bind'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_()).toEqual([2, 10, 15])
})
