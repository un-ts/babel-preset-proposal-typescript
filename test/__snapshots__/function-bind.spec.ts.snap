// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`function-bind 1`] = `"Cannot read properties of undefined (reading 'weight')"`;
